src/mosquito_emergence.cpp

```diff
@@ -36,7 +37,11 @@ Rcpp::XPtr<process_t> create_mosquito_emergence_process_cpp(
             }
             auto source = api.get_state(mosquito, unborn);
             if (source.size() < n) {
-                Rcpp::stop("Not enough mosquitos. Please raise mosquito_limit");
+                std::stringstream m;
+                m << "Not enough mosquitos (short by ";
+                m << n - source.size();
```

Comment:
Not sure it would be obvious to the user where to go to fix this issue if it occured. Maybe the message could point them towards  `parameters$mosquito_limit`,  and/or give some indication that checking the input seasonality (and EIR?) might be worthwhile?